source/functions.cpp

```diff
@@ -218,27 +146,27 @@ int moveFile(std::string source, std::string output) {
 
 
 int writeLocalisation(std::string file, std::string name, std::string description) {
-	return writeFile(file, "\n\t" + name + ":0 \"" + description + "\"", "a");
+	return writeFile(file, "\n\t" + name + ":0 \"" + description + "\"");
 }
 
 
 int convertToDds(std::string input, std::string output) {
 	int w, h, channels;
 	
-	unsigned char* png = SOIL_load_image(input.c_str(), &w, &h, &channels, SOIL_LOAD_AUTO);
+	unsigned char *png = SOIL_load_image(input.c_str(), &w, &h, &channels, SOIL_LOAD_AUTO);
```

Comment:
The project follows the format of `<variable type>[pointer symbol] <variable name>`. As such, just change this to `unsigned char*`

Reply:
Note: best to revert this change to fix the conflicting changes.